ogr/services/github/project.py

```diff
@@ -179,6 +180,33 @@ def get_branches(self) -> List[str]:
     def get_description(self) -> str:
         return self.github_repo.description
 
+    def add_user(self, user: str, access_level: AccessLevel):
+        """
+        AccessLevel.pull => Pull
+        AccessLevel.triage => Triage
+        AccessLevel.push => Push
+        AccessLevel.admin => Admin
+        AccessLevel.maintain => Maintain
+        """
+        access_dict = {
+            AccessLevel.pull: "Pull",
+            AccessLevel.triage: "Triage",
+            AccessLevel.push: "Push",
+            AccessLevel.admin: "Admin",
+            AccessLevel.maintain: "Maintain",
+        }
+        try:
+            invitation = self.github_repo.add_to_collaborators(
+                user, permission=access_dict[access_level]
+            )
+        except Exception:
+            raise GithubAPIException("User {user} not found")
+
+        if invitation is None:
+            raise GithubAPIException("User already added")
+
+        return None
```

Comment:
Is this needed?

Maybe we can add `-> None` type annotation to the function header to clearly state that we don't return anything.